api/src/environment.js

```diff
@@ -2,8 +2,13 @@ const url = require('url'),
       logger = require('./logger'),
       { COUCH_URL, UNIT_TEST_ENV } = process.env;
 
+let currentDeployInfo;
+const deployInfo = newDeployInfo => currentDeployInfo = newDeployInfo !== undefined ? newDeployInfo : currentDeployInfo;
+
 if (UNIT_TEST_ENV) {
-  module.exports = {};
+  module.exports = {
+    deployInfo
```

Comment:
Because we're exposing this for both prod and testing let's just add it to the exports outside of the if/else block. For example, at the bottom of the file, add this:

```js
module.exports.setDeployInfo = newDeployInfo => {
  if (newDeployInfo) {
    currentDeployInfo = newDeployInfo;
  }
};
```

Reply:
Done!